tests/unit/test_signers.py

```diff
@@ -273,7 +273,7 @@ def test_generate_presigned_post(self):
         self.auth.assert_called_with(
             credentials=self.credentials, region_name='region_name',
             service_name='signing_name')
-        self.add_auth.assert_called_once()
+        self.add_auth.call_count == 1
```

Comment:
They are necessary if we upgrade mock. assert_called_once() isn't actually a supported method on any version of mock. In mock 1.0, it will just act like any other mock method and return a new Mock object.

mock 1.1 added an 'unsafe' flag (defaults to False), which raises assertions if you try and call non-existent assert_ methods.